### PR TITLE
[FAB-18319] Ch.Part.API: IT Remove and rejoin channel from orderer

### PR DESCRIPTION
**Pull request by stephyee**
#### Type of change

- Test update

#### Description

Test removal of a channel from an orderer and join it back again to the same orderer.

#### Related issues

[FAB-18319](https://jira.hyperledger.org/browse/FAB-18319)

### CODE REVIEW

**Comment by wlahti** [comment]
A general comment... why isn't this just added on to the existing test you used as the starting point for this? Seems like this PR would be about 30 lines instead of 155 and would be easier to review that way.

File: integration/raft/channel_participation_test.go [hunk 1]
```diff
@@ -731,6 +731,161 @@ var _ = Describe("ChannelParticipation", func() {
...
+			}, network.EventuallyTimeout).Should(Equal(expectedChannelInfo))
+
+			submitPeerTxn(orderer3, peer, network, channelparticipation.ChannelInfo{
+				Name:              "participation-trophy",
+				URL:               "/participation/v1/channels/participation-trophy",
+				Status:            "active",
+				ConsensusRelation: "consenter",
+				Height:            9,
+			})
+		})
```
Comment (tock-ibm) [inline integration/raft/channel_participation_test.go]: Testing that onboarding completes should be performed against orderer1, which just joined, right?
Then test orderer1 is a follower, picking up the latest block, turn orderer1 into a consenter (member), and submit a tx to it as well?

Comment (tock-ibm) [reply integration/raft/channel_participation_test.go]: BTW, is there a test case where we delete a channel from an orderer while it is a consenter, not a follower? i.e first delete the channel from ordererX, and only then remove ordererX from the consenters set of that channel?

Comment (stephyee) [reply integration/raft/channel_participation_test.go]: As @wlahti noted, this test is based off another and I forget to switch to orderer1 instead as he suggested I'm going to add to the existing test. Also no, we don't have a test where we delete while the orderer is a consenter.

File: integration/raft/channel_participation_test.go [hunk 2]
```diff
@@ -731,6 +731,161 @@ var _ = Describe("ChannelParticipation", func() {
 				Height:            7,
 			})
 		})
+
+		It("joins application channels from genesis block and removes a channel using the channel participation API", func() {
```
[the remark this answers is not in the document]
Comment (stephyee) [reply integration/raft/channel_participation_test.go]: I missed that. Now, I've added the rejoin logic to an existing test.

File: integration/raft/channel_participation_test.go [hunk 3]
```diff
@@ -731,6 +776,7 @@ var _ = Describe("ChannelParticipation", func() {
 				Height:            7,
 			})
 		})
+
```
Comment (wlahti) [inline integration/raft/channel_participation_test.go]: Super nit that doesn't need to be addressed: I guess your editor settings like to format in this extra line?

**Review by wlahti — DISMISSED** [review]
Looks good to me!

**Review by tock-ibm — DISMISSED** [review]

File: integration/raft/channel_participation_test.go (hunk 3 above)
Comment (stephyee) [reply integration/raft/channel_participation_test.go]: or me not sure 😃

**Review by tock-ibm — APPROVED** [review]

**Review by Qdigital — APPROVED** [review]

**Review by C0rWin — APPROVED** [review]

**Pull request merged**